### PR TITLE
PPF-563 udb organizers

### PR DESCRIPTION
**Pull request by JonasVHG**
### Added

- `2024_08_06_104928_rename_organizers`: Added Migration to rename table in database.

### Changed

- rename `AddOrganizer` to `AddUdbOrganizer`
- rename `GetOrganizersRequest` to `GetUdbOrganizersRequest`
- rename `Organizer` to `UdbOrganizer`
- rename `OrganizerController` to `UdbOrganizerController`
- rename `OrganizerMapper` to `UdbOrganizerMapper`
- rename `OrganizerModel` to `UdbOrganizerModel`
- rename `OrganizerPolicy` to `UdbOrganizerPolicy`
- rename `OrganizerRepository` to `UdbOrganizerRepository`
- rename `Resource Organizer` to `UdbOrganizer`
- rename `UpdateIntegrationOrganizersRequest` to `UpdateIntegrationUiTdata… `
- rename `organizers` to `udbOrganizers`
- rename `withOrganizers` to `withUdbOrganizers`

### Fixed

- `UpdateOrganizationTest`:  typo mixed up organization organizer

---
Ticket: https://jira.publiq.be/browse/PPF-563

### CODE REVIEW

**Review by grubolsch — APPROVED** [review]

File: app/Domain/Integrations/Controllers/IntegrationController.php [hunk 1]
```diff
@@ -64,20 +64,20 @@
 final class IntegrationController extends Controller
 {
     public function __construct(
-        private readonly SubscriptionRepository $subscriptionRepository,
-        private readonly IntegrationRepository $integrationRepository,
-        private readonly IntegrationUrlRepository $integrationUrlRepository,
-        private readonly ContactRepository $contactRepository,
+        private readonly SubscriptionRepository         $subscriptionRepository,
```
Comment (LucWollants) [inline app/Domain/Integrations/Controllers/IntegrationController.php:67]: Some unwanted spacing is applied here by PHPStorm.

File: app/Domain/Integrations/Models/UiTdatabankOrganizerModel.php [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
 
-use App\Domain\Integrations\Organizer;
+use App\Domain\Integrations\UiTdatabankOrganizer;
 use App\Models\UuidModel;
 use Ramsey\Uuid\Uuid;
 
-final class OrganizerModel extends UuidModel
+final class UiTdatabankOrganizerModel extends UuidModel
 {
     protected $table = 'organizers';
```
Comment (LucWollants) [inline app/Domain/Integrations/Models/UiTdatabankOrganizerModel.php:13]: Should the table name also match the concept better?

File: app/Nova/Resources/UdbOrganizer.php [hunk 3]
```diff
@@ -13,11 +12,11 @@
...
 
 /**
- * @mixin OrganizerModel
+ * @mixin UdbOrganizer
  */
-final class Organizer extends Resource
+final class UdbOrganizer extends Resource
 {
-    public static string $model = OrganizerModel::class;
+    public static string $model = UdbOrganizer::class;
```
Comment (LucWollants) [inline app/Nova/Resources/UdbOrganizer.php:19]: I think this breaks the Nova view, it should be `UdbOrganizerModel`, this also needs changing on line 15

File: app/Nova/Resources/Integration.php [hunk 4]
```diff
@@ -222,7 +222,7 @@ function (Text $field, NovaRequest $request, FormData $formData) {
         }
 
         return array_merge($fields, [
-            HasMany::make('UDB3 Organizers', 'organizers', UdbOrganizer::class)
+            HasMany::make('UDB3 Organizers', 'organizers', UdbOrganizerModel::class)
```
Comment (LucWollants) [inline app/Nova/Resources/Integration.php:225]: I think the second parameter can be `udbOrganizers` which then matches the renamed method in the `IntegrationModel`

**Review by LucWollants — APPROVED** [review]

**Pull request merged**